core/org.wso2.carbon.ui/src/main/java/org/wso2/carbon/ui/transports/fileupload/JarZipUploadExecutor.java

```diff
@@ -15,22 +15,12 @@
  */
 package org.wso2.carbon.ui.transports.fileupload;
```

Comment:
We can remove this class completly since this is a new release

Reply:
Will be removed in another PR